packages/api/src/command/medical/patient/consolidated-get.ts

```diff
@@ -269,6 +270,7 @@ export async function getConsolidated({
 
     analytics(defaultAnalyticsProps);
 
+    bundle = deduplicateSearchSetBundle(bundle);
```

Comment:
are we turning this on by default for all customers, or hiding this behind a FF? 
My view is this should be for everyone and we can leave this as is. Imo dedup is a basic quality improvement that everyone should want and benefits from

Reply:
Yea, I don't think we should be introducing FF for this. However, it could be good to do that just to have a controlled rollout to our cx to see how they react. Thoughts?

Reply:
In line with this [here](https://metriport.slack.com/archives/C04DMKE9DME/p1723664928689969), I think we are going to take the FF approach ...

Reply:
Done